### ISSUE TITLE
Reload/restart IronScheme

### ISSUE THREAD

**Issue by ijacquez** [IronScheme/IronScheme#76]
@leppie,

I'm running IronScheme under Unity3D, non-gameplay mode (important), and all is well.

Specifically, I have a library that I load
```
var myProc = RuntimeExtensions.EvalWithEnvironment<Callable>("my-proc", "(environment '(my-lib))");
myProc.Call(...);
```
This works perfectly.

However, if I update `my-lib.sls`, I run the above (same instance), and the changes aren't applied, but only when I recompile do I see the changes.

My guess is that I'm modifying a static instance.

Is it possible to force reload/restart state in a way that I can make changes to `my-lib.sls`?

**Comment by leppie** [comment]
The closest you can get is to use something like this in the `interaction-environment`

```
>
> (library (test)
.   (export main)
.   (import (rnrs))
. (define (main)
.   (display "masssssin\n")))
> (import (test))
> (main)
masssssin
> (library (test)
.   (export main)
.   (import (rnrs))
. (define (main)
.   (display "massssssssssssssssssssin\n")))
> (main)
Unhandled exception during evaluation:
&who: main
&message: "identifier out of context"
&syntax:
  form: main
  subform: #f
&trace: #<syntax main>

> (import (test))
> (main)
massssssssssssssssssssin
>
```

You could use `include` to 'pull' the content from a file ie `(include "test.sls")` and then call import after. 

Unfortunately, the best I can offer for now.

**Comment by ijacquez** [comment]
Thank you for the suggestion. This is a good solution. I tried `load` previously (I failed to mention this in the issue), and with other Scheme implementations, this worked fine, but it didn't with IronScheme. Either way, it'll do.

Is there documentation to better understand how IronScheme is structured? Was my assumption correct that IronScheme is running as a static instance? I thought this over, and sometimes it's desirable to force reset the interpreter even if there aren't any changes in my library. Is it then possible to run many individual instances of IronScheme?
